### DISCUSSION TITLE
Getting started, need a tip on modifying default config for edgy.nvim

### DISCUSSION THREAD

**Discussion by whazlewo** [LazyVim/LazyVim#2515] [Q&A] [answered]
Greetings all,

I'm slowly getting my head around NeoVim/LazyVim. Some of the muscle memory is finally starting to set in, and now i'm experimenting with the LazyExtras. I like what edgy offers, but I was looking for a tip on where to begin figuring out how to move Aerial over to the left side of the screen under Neo-Tree. 

I'm currently trying to copy a bit of an answer I found in this forum, but its just not working. It reveals the symbols over to the left, but its buggy and just closes the plugin all together if i click it.  Any tips on how to just get the symbols broser over under where Neo-tree lives would be most appriciated :-)

 ```
plugins/ui.lua

return {
  {
      "folke/edgy.nvim",
      opts = function(_, opts)
        opts.exit_when_last = true
  
        -- change the default Explorer open action to use the new keymapping
        -- from our Neotree configuration above.
        opts.left[1].open = function()
          vim.api.nvim_input("<esc><space>ee")
        end

        -- add symbols section at the end
        table.insert(opts.left, #opts.left, {
          title = "Neo-Tree Symbols",
          ft = "neo-tree",
          filter = function(buf)
            return vim.b[buf].neo_tree_source == "document_symbols"
          end,

        pinned = true,
        open = function()
          vim.api.nvim_input("<esc><space>eo")
        end,
      })

      return opts
    end,
    },
}
```

Update
I think I may be a bit confused on something else as i dig into this. Is there a difference between the "symbols outline" and "Aerial" plugins?

Thank you in advance.

**Comment by jyuan0** [comment]
> Is there a difference between the "symbols outline" and "Aerial" plugins?

They're both code outline plugins, but assuming you copied from https://github.com/LazyVim/LazyVim/discussions/1203, the symbols you're seeing on the left are actually from [neo-tree](https://github.com/nvim-neo-tree/neo-tree.nvim).

> I'm currently trying to copy a bit of an answer I found in this forum, but its just not working.

If you copied from the discussion linked above, you also need to copy the neo-tree changes.

----

Update: If you're just trying to move Aerial to the left in an edgy window, I think you might be able to do that directly in an edgy plugin config, but I'm not sure if you're trying to move Aerial or neo-tree document symbols.

**Reply by jyuan0** [reply, marked as answer]
Maybe this is what you want (untested):
```lua
return {
  {
    "folke/edgy.nvim",
    opts = function(_, opts)
      table.insert(opts.left, {
        title = "Aerial",
        ft = "aerial",
        pinned = true,
        open = "AerialOpen",
      })
    end,
  },
}
```

**Reply by whazlewo** [reply]
I think this did exactly what I wanted. I just wanted that collection of stuff over in the other neo-tree section. I'm very slowly starting to get my head around this. Think you very much for this answer!
